plugins/flatpak/gs-flatpak.c

```diff
@@ -1665,9 +1665,18 @@ gs_flatpak_add_updates_pending (GsFlatpak *self, GsAppList *list,
 
 		update_priority = get_app_update_priority (app);
 
-		main_app = get_real_app_for_update (self, app, cancellable, error);
-		if (main_app == NULL)
-			return FALSE;
+		main_app = get_real_app_for_update (self, app, cancellable, &error_local);
```

Comment:
In the commit messages there are a few typos:
- `it whole` -> `the whole`
- `updage` -> `update`

Reply:
It was late. Sorry...